docs/reference_pickerfield.md

```diff
@@ -0,0 +1,142 @@
+
+#### `value`
+
+| Type   | Required               |
+| ------ | ---------------------- |
+| string | No (defaults to blank) |
+
+The value of the picker field. This string gets rendered into the string and can be edited by the user. Set this value in the XML to pre-populate the picker field.
+
+#### `placeholder`
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+A label that appears within the picker field. The placeholder only appears when the field is empty.
+
+#### `done-label`
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+This label appears in the button of the iOS modal. Pressing the button will apply the selected value to the field. Defaults to "Done".
+
+#### `cancel-label`
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+This label appears in the button of the iOS modal. Pressing the button will ignore the selected value, and the value in the picker field will remain unchanged.
+
+#### `field-style`
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+A space-separated list of styles to apply to the picker field container. This is the visual element that surrounds the selected picker value. You can use these styles to define borders, margin, and padding around the picker field. See [Styles](/docs/reference_style). Note that text style rules cannot be applied to the container.
+
+#### `field-text-style`
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+A space-separated list of styles to apply to the text value of the picker field. Use these styles to define the font size, color, and weight of the value in the picker field. See [Styles](/docs/reference_style). Note that only text style rules can be applied to the text.
+
+#### `modal-style`
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+A space-separated list of styles to apply to the iOS bottom-sheet modal. You can use these styles to define borders, margin, and padding around the bottom modal that appears on iOS. On Android, the system defines the look of the modal, so these styles will not be applied. See [Styles](/docs/reference_style). Note that text style rules cannot be applied to the modal.
+
+#### `modal-text-style`
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+A space-separated list of styles to apply to the buttons in the iOS bottom-sheet modal. You can use these styles to define the size, color, and weight of the buttons in the iOS modal. On Android, the system defines the look of the modal, so these styles will not be applied. See [Styles](/docs/reference_style). Note that text style rules cannot be applied to the modal.
+
+#### `id`
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+A global attribute uniquely identifying the element in the whole document.
```

Comment:
When would we need to provide an `id`? Probably a small explanation would help engineers.